src/charm.py

```diff
@@ -604,6 +611,31 @@ def _push_to_workload(self, filename, content, event):
             LOGGER.info("workload container not ready - deferring")
             event.defer()
 
+    def _update_trusted_ca_certs(self, container: Container) -> bool:
+        """Update trusted CA certificates with the cert from configuration.
+
+        Livepatch needs to restart to use newly received certificates.
+
+        Args:
+            container (Container): The workload container, the caller must ensure that we can connect.
+
+        Returns:
+            bool: A boolean to indicate whether the workload service should be restarted.
+        """
+        if self.config.get("contracts.ca"):
+            LOGGER.error("pushing ca cert")
+            cert = b64decode(self.config.get("contracts.ca")).decode("utf8")
+            container.push(TRUSTED_CA_FILENAME, cert, make_dirs=True)
+            LOGGER.error("updating ca cert")
+            stdout, stderr = container.exec(["update-ca-certificates", "--fresh"]).wait_output()
+            LOGGER.info("stdout update-ca-certificates: %s", stdout)
+            LOGGER.info("stderr update-ca-certificates: %s", stderr)
+
+            return True
+
+        LOGGER.error("ca config not set")
+        return False
```

Comment:
**suggestion**: can you please change it to a guard-clause?

```python
if not self.config.get(...)
    LOGGER.error("ca config not set")
    return False

# ...
```

Reply:
done